### PR TITLE
krew: Rename kubectl-az to kubectl-aks

### LINKED ISSUE #30: Rename plugin to `kubectl-aks`
After discussing with the `krew`'s maintainers, we decided to change the name from `kubectl-az` to `kubectl-aks` as it describes better the scope of the plugin. It implies that we have to change the repo, the binaries and make a new release.

Another request from the `krew`'s maintainers is to improve the description, see full discussion in https://github.com/kubernetes-sigs/krew-index/pull/3052.

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Fixes #30 

## Todo

Before Merge: 
- [x] Setup the kubectl-aks-ci-rg for CI
- [x] Green CI Run

After Merge: 
- [ ] Clean up kubectl-az-ci-rg
- [ ] Rename the Repository

## Testing Done

```bash
→ make integration-test 
export GO111MODULE=on CGO_ENABLED=0 && \
export GOOS=linux GOARCH=amd64 && \
go build -ldflags "-X github.com/Azure/kubectl-aks/cmd.version=`git describe --tags --always`-dirty -extldflags '-static'" \
        -o kubectl-aks-${GOOS}-${GOARCH} \
        github.com/Azure/kubectl-aks
mv kubectl-aks-linux-amd64 kubectl-aks
KUBECTL_AKS="/home/qasim/work/github.com/Azure/kubectl-az/kubectl-aks" \
        go test -v ./test/integration/... -integration
Running integration tests
=== RUN   TestCheckAPIServerConnectivity
    integration_test.go:36: Running command: /home/qasim/work/github.com/Azure/kubectl-az/kubectl-aks --node aks-agentpool-77471288-vmss00001x check-apiserver-connectivity
    integration_test.go:36: Command output: 
        Running...
        
        Connectivity check: succeeded
--- PASS: TestCheckAPIServerConnectivity (24.09s)
=== RUN   TestRunCommand
    integration_test.go:42: Running command: /home/qasim/work/github.com/Azure/kubectl-az/kubectl-aks --node aks-agentpool-77471288-vmss00001x run-command echo test
    integration_test.go:42: Command output: 
        Running...
        
        [stdout]
        test
        
        [stderr]
    integration_test.go:48: Running command: /home/qasim/work/github.com/Azure/kubectl-az/kubectl-aks --node aks-agentpool-77471288-vmss00001x run-command echo test >&2
    integration_test.go:48: Command output: 
        Running...
        
        [stdout]
        
        [stderr]
        test
--- PASS: TestRunCommand (47.52s)
PASS
ok      github.com/Azure/kubectl-aks/test/integration   71.615s

```

### CODE REVIEW

File: .krew.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: az
```
Comment (blanquicet) [inline .krew.yaml:4]:
```suggestion
  name: aks
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Microsoft Azure CLI kubectl plugin
+# Microsoft AKS CLI kubectl plugin
```
Comment (blanquicet) [inline README.md:1]:
```suggestion
# Microsoft Azure Kubernetes Service (AKS) CLI kubectl plugin
```
WDYT?

**Review by blanquicet — CHANGES_REQUESTED** [review]
It looks fine. IMO, only the krew plugin name is missing.

It is also missing to improve the description as requested by the krew maintainer.

File: README.md (hunk 2 above)
Comment (mqasimsarfraz) [reply README.md:1]: Agreed. Makes sense to use the complete name here.

**Comment by mqasimsarfraz** [comment]
> It is also missing to improve the description as requested by the krew maintainer.

Oops, yeah I missed that. I have updated the PR to include the latest description.

**Comment by mqasimsarfraz** [comment]
Thanks for the review. I will rename the repository and merge the PR. Hopefully, no issues! :crossed_fingers:

**Pull request merged**